packages/auth/src/tests/app.ts

```diff
@@ -1,4 +1,4 @@
-import Knex from 'knex'
+import { Knex } from 'knex'
```

Comment:
It seems like the import you've been doing in other places is in lowercase, does this import still work?

Reply:
yes unfortunately, see: https://github.com/knex/knex/blob/master/UPGRADING.md#upgrading-to-version-0950